qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -92,6 +92,8 @@ function cleanup()
 
     # cleanup/reset default pool
     rbd config global rm global rbd_default_pool
+    # cleanup/reset discard granularity bytes
+    rbd config global rm client rbd_discard_granularity_bytes
```

Comment:
nit: the comment "cleanup/reset default pool" referred also to `ceph osd pool delete` command below. So I would put your command below it. Or just have one more generic comment about env cleanup (if we need it).